Dockerfile

```diff
@@ -60,9 +67,9 @@ RUN set -e \
     && useradd -d /data zenith \
     && chown -R zenith:zenith /data
 
-COPY --from=build --chown=zenith:zenith /home/nonroot/target/release/pageserver /usr/local/bin
-COPY --from=build --chown=zenith:zenith /home/nonroot/target/release/safekeeper /usr/local/bin
-COPY --from=build --chown=zenith:zenith /home/nonroot/target/release/proxy      /usr/local/bin
+COPY --from=build --chown=zenith:zenith /home/nonroot/pageserver /usr/local/bin
```

Comment:
To me that looks an extra `cp` and swapping one conventional directory to another one, would that make sense to keep things as is?

Reply:
Cannot leave it as is, because `--mount` directive is valid during the single `RUN` only, and contents of it are not observed by the build machinery after this `RUN` has successfully finished.